package.json

```diff
@@ -36,24 +33,31 @@
     "@lavamoat/allow-scripts": "^2.0.2",
     "@metamask/auto-changelog": "^2.5.0",
     "@metamask/eslint-config": "^9.0.0",
+    "@metamask/eslint-config-jest": "^9.0.0",
     "@metamask/eslint-config-nodejs": "^9.0.0",
     "@metamask/eslint-config-typescript": "^9.0.1",
+    "@types/jest": "^27.4.1",
     "@types/json-rpc-random-id": "^1.0.1",
+    "@types/node": "^17.0.23",
     "@types/pify": "^5.0.1",
     "@typescript-eslint/eslint-plugin": "^4.20.0",
     "@typescript-eslint/parser": "^4.20.0",
     "eslint": "^7.23.0",
     "eslint-config-prettier": "^8.1.0",
+    "eslint-import-resolver-typescript": "^2.7.1",
     "eslint-plugin-import": "^2.22.1",
+    "eslint-plugin-jest": "^24.1.3",
     "eslint-plugin-jsdoc": "^36.1.0",
     "eslint-plugin-node": "^11.1.0",
     "eslint-plugin-prettier": "^3.3.1",
-    "ganache-core": "^2.13.2",
+    "events": "^3.3.0",
```

Comment:
This dependency is not actually being used. `events` is a built-in Node.js API, we don't need this dependency.

Reply:
Oops, good catch!